app/controllers/mixins/custom_buttons.rb

```diff
@@ -13,6 +13,8 @@ def custom_toolbar_explorer
         Mixins::CustomButtons::Result.new(:single)
       elsif @lastaction == "show_list"
         Mixins::CustomButtons::Result.new(:list)
+      elsif x_tree[:tree] == :rbac_tree
+        @lastaction == "show_list" ? Mixins::CustomButtons::Result.new(:list) : Mixins::CustomButtons::Result.new(:single)
```

Comment:
this is just `Mixins::CustomButtons::Result.new(:single)`

(the show_list branch will never happen because it will match on line 14)